### ISSUE TITLE
Find people and groups not working

### ISSUE THREAD

**Issue by him229** [rsms/fb-mac-messenger#374]
`Find people and groups...` does not work on sierra. No error messages, just does nothing. Even keyboard shortcut ⌘F doesn't work. Running v0.1.12 @rsms

**Comment by Goles** [comment]
I can confirm this in El Capitan `10.11.6` too.

**Comment by rsms** [comment]
Good catch. Just facebook that changed the HTML and now we need to update the main.js bindings ...again. This is low priority. You can click the search box. ("Find people and groups..." just automates clicking it)

**Comment by him229** [comment]
Potential fix https://github.com/rsms/fb-mac-messenger/pull/381 @rsms
